.github/workflows/test-r-scripts.yaml

```diff
@@ -63,13 +76,27 @@ jobs:
           r-version: ${{ matrix.config.r }}
           http-user-agent: ${{ matrix.config.http-user-agent }}
 
+      - name: Install system dependency of usethis >= 2.0.0
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt install libgit2-dev
```

Comment:
Even is we do not depend on usethis directly, it may be installed via devtools -- so we need the right system dependencies.

Reply:
I expect the workflows will adapt to usethis 3 because the same people develop both. For now, I had to figure it out for myself and was a real pain.